tests/functionals_tests/test_moment_coefficient.py

```diff
@@ -36,7 +34,25 @@ def test2(self):
 
         group.connect("indep_var_comp.S_ref_total", "moment_calc.S_ref_total")
 
-        run_test(self, group)
+        run_test(self, group, complex_flag=True, method="cs")
+
+    def test3(self):
```

Comment:
I don't have a strong opinion, but is there any reason you setup `test3` separately from `test2`? Maybe these two can be merged into one?

Reply:
Done